src/components/organisms/TriggersFormItems/ResourceTriggerSelect/ResourceTriggerSelect.tsx

```diff
@@ -0,0 +1,85 @@
+import {useContext, useMemo} from 'react';
+
+import {Form, Select} from 'antd';
+
+import {ReactComponent as TestSuitesIcon} from '@assets/test-suites-icon.svg';
+
+import {ExecutorIcon} from '@atoms';
+
+import {MainContext} from '@contexts';
+
+import {Text} from '@custom-antd';
+
+import {useAppSelector} from '@redux/hooks';
+import {selectExecutors} from '@redux/reducers/executorsSlice';
+import {getTestExecutorIcon} from '@redux/utils/executorIcon';
+
+import {useGetAllTestSuitesQuery} from '@services/testSuites';
+import {useGetAllTestsQuery} from '@services/tests';
+
+import Colors from '@styles/Colors';
+
+import {required} from '@utils/form';
+
+import {StyledResourceOptionWrapper} from './ResourceTriggerSelect.styled';
+
+const {Option, OptGroup} = Select;
+
+const ResourceTriggerSelect = () => {
+  const {isClusterAvailable} = useContext(MainContext);
+
+  const executors = useAppSelector(selectExecutors);
+
+  const {data: tests = []} = useGetAllTestsQuery(null, {skip: !isClusterAvailable});
+  const {data: testSuites = []} = useGetAllTestSuitesQuery(null, {
+    skip: !isClusterAvailable,
+  });
+
+  const testsData = useMemo(() => {
+    return tests.map(item => ({
+      name: item.test.name,
+      namespace: item.test.namespace,
+      type: item.test.type,
+    }));
+  }, [tests]);
+
+  const testSuitesData = useMemo(() => {
+    return testSuites.map(item => ({
+      name: item.testSuite.name,
+      namespace: item.testSuite.namespace,
+    }));
+  }, [testSuites]);
+
+  return (
+    <Form.Item label="Testkube resource" required name="testNameSelector" rules={[required]}>
```

Comment:
It can't be. I would like it to be just `Select` too, but without `Form.Item` in this component, it doesn't really work. Default value not provided to select, as well as `isFieldsTouched` not triggered when I select something else.

Reply:
* The default value doesn't have to be provided - there is no value, then it's default. Additionally, you may pass any custom props you want to the input.
* `isFieldsTouched` will be updated when you update the value. When you handle the `onChange` prop in your custom input, any call to it will just make the field touched and the value updated.

Reply:
I might not get your point here. Isn't it more simple to leave it as it is? If I remove FormItem I would have to Create custom label, required mark, control the value with `onChange` and `value` props. What is the benefits of this approach, or I didn't get your point correctly?

Reply:
You don't have to add custom labels or required marks. The `Form.Item` should be kept, just a level above - in the form where `ResourceTriggerSelect` is actually used.

The `onChange` and `value` will need to be controlled though, but it's just a few lines of a common code and will make it *much* simpler for unit tests and maintenance.